irohad/main/application.cpp

```diff
@@ -139,7 +143,11 @@ Irohad::Irohad(
 #if defined(USE_BURROW)
         vm_caller_ = std::make_unique<iroha::ametsuchi::BurrowVmCaller>();
 #endif
-        return initStorage(startup_wsv_data_policy);
+        return initStorage(startup_wsv_data_policy,
+                           config_.database_config
+                                   && config_.database_config->type == "rocksdb"
```

Comment:
MUST be postgres or roksdb no other value acceptable! Provide full-text descripted error message to user.

Reply:
This is not a validation part of code.

Reply:
I see. This is not a validation part, that validation part simply not implemented )